syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -381,6 +381,8 @@ def pow(self, power):
         Args:
             power (int): the exponent supposed to be an integer > 0
         """
+        if power < 0:
+            raise RuntimeError("Integers to negative integer powers are not allowed.")
```

Comment:
```suggestion
            raise RuntimeError("Negative integer powers are not allowed.")
```
Missed this one :sweat_smile:

Reply:
I applied this 👍